azerothcore/azerothcore-wotlk
# fix(CORE/Commands): .Pinfo

##### TESTS PERFORMED:
Yes, imported without errors. Checked and tested in game, everything looks fine.

##### HOW TO TEST THE CHANGES:
1: .guild create "Guild Name"
2: .Pinfo
3: Now you can see Guild Name Guild Rank Note Office Note And Xp - Total Xp

##### Target branch(es):
Master
